Flaky gradle build failures

I'm not sure if these failures are related or not. They are all failure during gradle build:

* https://ci.chromium.org/ui/p/flutter/builders/prod/Mac_android%20textfield_perf__timeline_summary/563/overview

```
stderr: [+7386 ms] FAILURE: Build failed with an exception.
stderr: [        ] * What went wrong:
stderr: [        ] A problem occurred configuring root project 'android'.
stderr: [        ] > Could not resolve all artifacts for configuration ':classpath'.
stderr: [        ]    > Could not download fastutil-7.2.0.jar (it.unimi.dsi:fastutil:7.2.0)
stderr: [        ]       > Could not get resource 'https://jcenter.bintray.com/it/unimi/dsi/fastutil/7.2.0/fastutil-7.2.0.jar'.
stderr: [        ]          > Premature end of Content-Length delimited message body (expected: 17,309,408; received: 1,048,576)
stderr: [        ] * Try:
stderr: [        ] Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
stderr: [        ] * Get more help at https://help.gradle.org
stderr: [        ] BUILD FAILED in 17s
stdout: [+1769 ms] Running Gradle task 'assembleProfile'... (completed in 19.7s)
stdout: [   +2 ms] "flutter drive" took 36,644ms.
stderr: [   +3 ms] Gradle task assembleProfile failed with exit code 1
```

* https://ci.chromium.org/ui/p/flutter/builders/prod/Linux%20flutter_engine_group_performance/303/overview

```
stderr: FAILURE: Build failed with an exception.
stderr: 
stderr: * What went wrong:
stderr: Could not determine the dependencies of task ':app:lintVitalRelease'.
stderr: > Could not resolve all artifacts for configuration ':app:releaseRuntimeClasspath'.
stderr:    > Could not resolve io.flutter:arm64_v8a_release:1.0.0-20375cebcf3584a23e424e8a722806f77702fa85.
stderr:      Required by:
stderr:          project :app > project :flutter
stderr:       > Could not resolve io.flutter:arm64_v8a_release:1.0.0-20375cebcf3584a23e424e8a722806f77702fa85.
stderr:          > Could not get resource 'https://jcenter.bintray.com/io/flutter/arm64_v8a_release/1.0.0-20375cebcf3584a23e424e8a722806f77702fa85/arm64_v8a_release-1.0.0-20375cebcf3584a23e424e8a722806f77702fa85.pom'.
stderr:             > Could not GET 'https://jcenter.bintray.com/io/flutter/arm64_v8a_release/1.0.0-20375cebcf3584a23e424e8a722806f77702fa85/arm64_v8a_release-1.0.0-20375cebcf3584a23e424e8a722806f77702fa85.pom'.
stderr:                > Read timed out
stderr:    > Could not resolve io.flutter:x86_64_release:1.0.0-20375cebcf3584a23e424e8a722806f77702fa85.
stderr:      Required by:
stderr:          project :app > project :flutter
stderr:       > Could not resolve io.flutter:x86_64_release:1.0.0-20375cebcf3584a23e424e8a722806f77702fa85.
stderr:          > Could not get resource 'https://jcenter.bintray.com/io/flutter/x86_64_release/1.0.0-20375cebcf3584a23e424e8a722806f77702fa85/x86_64_release-1.0.0-20375cebcf3584a23e424e8a722806f77702fa85.pom'.
stderr:             > Could not GET 'https://jcenter.bintray.com/io/flutter/x86_64_release/1.0.0-20375cebcf3584a23e424e8a722806f77702fa85/x86_64_release-1.0.0-20375cebcf3584a23e424e8a722806f77702fa85.pom'.
stderr:                > Read timed out
stderr: 
stderr: * Try:
stderr: Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
stderr: 
stderr: * Get more help at https://help.gradle.org
stderr: 
stderr: BUILD FAILED in 3m 22s
stdout: 
stdout: Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
stdout: Use '--warning-mode all' to show the individual deprecation warnings.
stdout: See https://docs.gradle.org/6.5/userguide/command_line_interface.html#sec:command_line_warnings
"./gradlew" exit code: 1
```

* https://ci.chromium.org/ui/p/flutter/builders/prod/Linux%20basic_material_app_android__compile/831/overview

```
stderr: FAILURE: Build failed with an exception.
stdout: 
stderr: 
stdout: 
stderr: * What went wrong:
stdout: 
stderr: Could not determine the dependencies of task ':app:lintVitalRelease'.
stdout: 
stderr: > Could not resolve all artifacts for configuration ':app:debugCompileClasspath'.
stdout: 
stderr:    > Could not download kotlin-stdlib-jdk7-1.3.50.jar (org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.3.50): Skipped due to earlier error
stdout: 
stderr: 
stdout: 
stderr: * Try:
stdout: 
stderr: Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
stdout: 
stderr: 
stdout: 
stderr: * Get more help at https://help.gradle.org
stdout: 
stderr: 
stdout: 
stderr: BUILD FAILED in 2m 42s
stdout: Running Gradle task 'assembleRelease'...                          163.7s
stderr: Gradle task assembleRelease failed with exit code 1
```

--- a/dev/benchmarks/complex_layout/android/build.gradle
+++ b/dev/benchmarks/complex_layout/android/build.gradle
@@ -5,7 +5,13 @@
 buildscript {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 
     dependencies {
@@ -16,7 +22,13 @@ buildscript {
 allprojects {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 }
 

--- a/dev/benchmarks/macrobenchmarks/android/build.gradle
+++ b/dev/benchmarks/macrobenchmarks/android/build.gradle
@@ -5,7 +5,13 @@
 buildscript {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 
     dependencies {
@@ -16,7 +22,13 @@ buildscript {
 allprojects {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 }
 

--- a/dev/benchmarks/microbenchmarks/android/build.gradle
+++ b/dev/benchmarks/microbenchmarks/android/build.gradle
@@ -5,7 +5,13 @@
 buildscript {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 
     dependencies {
@@ -16,7 +22,13 @@ buildscript {
 allprojects {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 }
 

--- a/dev/benchmarks/multiple_flutters/android/build.gradle
+++ b/dev/benchmarks/multiple_flutters/android/build.gradle
@@ -7,7 +7,13 @@ buildscript {
     ext.kotlin_version = "1.3.72"
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
     dependencies {
         classpath "com.android.tools.build:gradle:4.1.2"
@@ -21,7 +27,13 @@ buildscript {
 allprojects {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 }
 

--- a/dev/benchmarks/platform_views_layout/android/build.gradle
+++ b/dev/benchmarks/platform_views_layout/android/build.gradle
@@ -5,7 +5,13 @@
 buildscript {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 
     dependencies {
@@ -16,7 +22,13 @@ buildscript {
 allprojects {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 }
 

--- a/dev/benchmarks/platform_views_layout_hybrid_composition/android/build.gradle
+++ b/dev/benchmarks/platform_views_layout_hybrid_composition/android/build.gradle
@@ -5,7 +5,13 @@
 buildscript {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 
     dependencies {
@@ -16,7 +22,13 @@ buildscript {
 allprojects {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 }
 

--- a/dev/benchmarks/test_apps/stocks/android/build.gradle
+++ b/dev/benchmarks/test_apps/stocks/android/build.gradle
@@ -5,7 +5,13 @@
 buildscript {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 
     dependencies {
@@ -16,7 +22,13 @@ buildscript {
 allprojects {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 }
 

--- a/dev/integration_tests/abstract_method_smoke_test/android/build.gradle
+++ b/dev/integration_tests/abstract_method_smoke_test/android/build.gradle
@@ -6,7 +6,13 @@ buildscript {
     ext.kotlin_version = '1.3.50'
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 
     dependencies {
@@ -18,7 +24,13 @@ buildscript {
 allprojects {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 }
 

--- a/dev/integration_tests/android_custom_host_app/build.gradle
+++ b/dev/integration_tests/android_custom_host_app/build.gradle
@@ -5,7 +5,13 @@
 buildscript {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:4.1.0'
@@ -15,7 +21,13 @@ buildscript {
 allprojects {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 }
 

--- a/dev/integration_tests/android_embedding_v2_smoke_test/android/build.gradle
+++ b/dev/integration_tests/android_embedding_v2_smoke_test/android/build.gradle
@@ -6,7 +6,13 @@ buildscript {
     ext.kotlin_version = '1.3.50'
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 
     dependencies {
@@ -18,7 +24,13 @@ buildscript {
 allprojects {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 }
 

--- a/dev/integration_tests/android_host_app_v2_embedding/build.gradle
+++ b/dev/integration_tests/android_host_app_v2_embedding/build.gradle
@@ -5,7 +5,13 @@
 buildscript {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:4.1.0'
@@ -15,7 +21,13 @@ buildscript {
 allprojects {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 }
 

--- a/dev/integration_tests/android_semantics_testing/android/build.gradle
+++ b/dev/integration_tests/android_semantics_testing/android/build.gradle
@@ -5,7 +5,13 @@
 buildscript {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 
     dependencies {
@@ -16,7 +22,13 @@ buildscript {
 allprojects {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 }
 

--- a/dev/integration_tests/android_views/android/build.gradle
+++ b/dev/integration_tests/android_views/android/build.gradle
@@ -5,7 +5,13 @@
 buildscript {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 
     dependencies {
@@ -16,7 +22,13 @@ buildscript {
 allprojects {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 }
 

--- a/dev/integration_tests/channels/android/build.gradle
+++ b/dev/integration_tests/channels/android/build.gradle
@@ -5,7 +5,13 @@
 buildscript {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 
     dependencies {
@@ -16,7 +22,13 @@ buildscript {
 allprojects {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 }
 

--- a/dev/integration_tests/external_ui/android/build.gradle
+++ b/dev/integration_tests/external_ui/android/build.gradle
@@ -5,7 +5,13 @@
 buildscript {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 
     dependencies {
@@ -16,7 +22,13 @@ buildscript {
 allprojects {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 }
 

--- a/dev/integration_tests/flavors/android/build.gradle
+++ b/dev/integration_tests/flavors/android/build.gradle
@@ -5,7 +5,14 @@
 buildscript {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 
     dependencies {
@@ -16,7 +23,13 @@ buildscript {
 allprojects {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 }
 

--- a/dev/integration_tests/flutter_gallery/android/build.gradle
+++ b/dev/integration_tests/flutter_gallery/android/build.gradle
@@ -5,7 +5,13 @@
 buildscript {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 
     dependencies {
@@ -16,7 +22,13 @@ buildscript {
 allprojects {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
         maven {
             url 'https://google.bintray.com/exoplayer/'
         }

--- a/dev/integration_tests/gradle_deprecated_settings/android/build.gradle
+++ b/dev/integration_tests/gradle_deprecated_settings/android/build.gradle
@@ -5,7 +5,13 @@
 buildscript {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 
     dependencies {
@@ -16,7 +22,13 @@ buildscript {
 allprojects {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 }
 

--- a/dev/integration_tests/hybrid_android_views/android/build.gradle
+++ b/dev/integration_tests/hybrid_android_views/android/build.gradle
@@ -5,7 +5,13 @@
 buildscript {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 
     dependencies {
@@ -16,7 +22,13 @@ buildscript {
 allprojects {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 }
 

--- a/dev/integration_tests/module_host_with_custom_build_v2_embedding/build.gradle
+++ b/dev/integration_tests/module_host_with_custom_build_v2_embedding/build.gradle
@@ -5,7 +5,13 @@
 buildscript {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:4.1.0'
@@ -15,7 +21,13 @@ buildscript {
 allprojects {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 }
 

--- a/dev/integration_tests/non_nullable/android/build.gradle
+++ b/dev/integration_tests/non_nullable/android/build.gradle
@@ -6,7 +6,13 @@ buildscript {
     ext.kotlin_version = '1.3.50'
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 
     dependencies {
@@ -18,7 +24,13 @@ buildscript {
 allprojects {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 }
 

--- a/dev/integration_tests/platform_interaction/android/build.gradle
+++ b/dev/integration_tests/platform_interaction/android/build.gradle
@@ -5,7 +5,13 @@
 buildscript {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 
     dependencies {
@@ -16,7 +22,13 @@ buildscript {
 allprojects {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 }
 

--- a/dev/integration_tests/release_smoke_test/android/build.gradle
+++ b/dev/integration_tests/release_smoke_test/android/build.gradle
@@ -5,7 +5,13 @@
 buildscript {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 
     dependencies {
@@ -16,7 +22,13 @@ buildscript {
 allprojects {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 }
 

--- a/dev/integration_tests/ui/android/build.gradle
+++ b/dev/integration_tests/ui/android/build.gradle
@@ -5,7 +5,13 @@
 buildscript {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 
     dependencies {
@@ -16,7 +22,14 @@ buildscript {
 allprojects {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 }
 

--- a/dev/manual_tests/android/build.gradle
+++ b/dev/manual_tests/android/build.gradle
@@ -6,7 +6,13 @@ buildscript {
     ext.kotlin_version = '1.3.50'
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 
     dependencies {
@@ -18,7 +24,13 @@ buildscript {
 allprojects {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        // TODO(jmagman): Remove with https://github.com/flutter/flutter/issues/78338
+        jcenter {
+            content {
+                includeModule("org.jetbrains.trove4j", "trove4j")
+            }
+        }
     }
 }
 

